src/routes/object/updateObject.ts

```diff
@@ -162,6 +162,7 @@ export default async function routes(fastify: FastifyInstance) {
       const { error: updateError, status: updateStatus } = await request.postgrest
         .from<Obj>('objects')
         .update({
+          updated_at: new Date().toISOString(),
```

Comment:
A `BEFORE UPDATE` SQL trigger would be more robust, so that the column would be updated regardless of where we update from. Would you be ok with adding a migration under `migrations/tenant` to do that?

Reply:
I guess, the SQL triggers are in preview for now, so I didn't check on that, 
And about "migration under `migrations/tenant` ",  Let me check with that if I can or not :)

Reply:
I meant something like this: https://www.revsys.com/tidbits/automatically-updating-a-timestamp-column-in-postgresql/.

Reply:
@thebengeu , I didn't get the `migrations/tenant` part, might need your help, if you can do that, that would be great, (I didn`t understand how it's integrated, needed bit more diging),  or 
You can update it later, once the SQL trigger becomes production ready. what you think !!

Reply:
I can write PSQL query

```
CREATE OR REPLACE FUNCTION update_modified_column() 
RETURNS TRIGGER AS $$
BEGIN
    NEW.updated_at = now();
    RETURN NEW; 
END;
$$ language 'plpgsql';

CREATE TRIGGER update_object_updated_time BEFORE UPDATE ON storage.objects FOR EACH ROW EXECUTE PROCEDURE  update_modified_column();
```

just check it once

Reply:
Looks good to me, just a couple of nits:

- `plpgsql` doesn't have to be in quotes.
- Should be `updated_at` instead of `modified`.
- Maybe `update_objects_updated_at` instead of `update_object_updated_time`?

Reply:
```sql
CREATE OR REPLACE FUNCTION update_updated_at_column() 
RETURNS TRIGGER AS $$
BEGIN
    NEW.updated_at = now();
    RETURN NEW; 
END;
$$ language plpgsql;

CREATE TRIGGER update_objects_updated_at BEFORE UPDATE ON storage.objects FOR EACH ROW EXECUTE PROCEDURE update_updated_at_column();
```

Hope that's good, can you suggest any name for it, so that I can add as that name

Reply:
Yeah that's great, either works!